Dockerfile

```diff
@@ -3,10 +3,10 @@ FROM python:3.10-slim
 WORKDIR /app
 
 COPY requirements.txt requirements.txt
-COPY app app
+
+# Copy the entire app folder inside the app folder in the workdir
+COPY /app app/
 
 RUN pip install --no-cache-dir -r requirements.txt
 
 EXPOSE 8000
-
-CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
I placed it in the docker compose so that the run was not executed when the container is built. Now the command is run after all the containers are up.